src/components/PhoneNumberInput/Lazy.js

```diff
@@ -0,0 +1,21 @@
+import { h, Component } from 'preact'
+import style from './style.css'
+
+const Loading = () => <div className={style.loading}>Loading...</div>
+
+class PhoneNumberInputLazy extends Component {
+  constructor(props){
+    super(props)
+    this.state = {component: Loading }
+    // We are loading the mobile validation component dynamically
+    // The code is splitted in two separate chunks.
+
+    import(/* webpackChunkName: "mobile-number-validations" */ './index.js').then(component => {
```

Comment:
I would call this `crossDevice`. Because it makes more sense in the future to package that as a bundle, rather than just the number validation.